test/unit/elasticsearch/elasticsearch_index_test.rb

```diff
@@ -7,80 +7,28 @@ class ElasticsearchIndexTest < MiniTest::Unit::TestCase
   include Fixtures::DefaultMappings
 
   def setup
-    @base_uri = URI.parse("http://example.com:9200")
-    search_config = SearchConfig.new
-    @wrapper = Elasticsearch::Index.new(@base_uri, "mainstream_test", "mainstream_test", default_mappings, search_config)
-
-    @traffic_index = Elasticsearch::Index.new(@base_uri, "page-traffic_test", "page-traffic_test", page_traffic_mappings, search_config)
-    PopularityLookup.any_instance.stubs(:traffic_index).returns(@traffic_index)
-    @traffic_index.stubs(:real_name).returns("page-traffic_test")
+    @index = build_mainstream_index
   end
 
-  def stub_popularity_index_requests(paths, popularity, total_pages=10, total_requested=total_pages, paths_to_return=paths)
-    # stub the request for total results
-    stub_request(:get, "http://example.com:9200/page-traffic_test/_search").
-            with(:body => { "query" => { "match_all" => {}}, "size" => 0 }.to_json).
-            to_return(:status => 200, :body => { "hits" => { "total" => total_pages }}.to_json)
-
-    # stub the search for popularity data
-    expected_query = {
-      "query" => {
-        "terms" => {
-          "path_components" => paths,
-        },
-      },
-      "fields" => ["rank_14"],
-      "sort" => [
-        { "rank_14" => { "order" => "asc" } }
-      ],
-      "size" => total_requested
-    }
-    response = {
-      "hits" => {
-        "hits" => paths_to_return.map {|path|
-          {
-            "_id" => path,
-            "fields" => {
-              "rank_14" => popularity
-            }
-          }
-        }
-      }
-    }
-
-    stub_request(:get, "http://example.com:9200/page-traffic_test/_search").
-            with(:body => expected_query.to_json).
-            to_return(:status => 200, :body => response.to_json, :headers => {})
-  end
-
-  def successful_response
-    <<-eos
-{"took":5,"items":[
-  { "index": { "_index":"mainstream_test", "_type":"edition", "_id":"/foo/bar", "ok":true } }
-]}
-    eos
-  end
+  # Elasticsearch::Index#real_name
```

Comment:
Ah - I think this is meant to indicate the start of tests for the "real_name" method of Index.  I've not seen that convention before - perhaps if it said `# Tests for Elasticsearch::Index#real_name` that would be clearer?  Or maybe this indicates that this file should be split up further?

Everything else in this PR looks :sparkles: so I'm going to merge, and if you'd like to split up the file that could be done later.  Or not.